pages/Javascript-API.md

```diff
@@ -44,6 +44,39 @@ Each object contains sub-objects which correlate to each element's subelements a
 
 If changes are made to the `config` object, you must call `scene.updateConfig()` for them to take effect.
 
+####`scene.config.textures`
```

Comment:
By "not requiring additional documentation", I mean that the `config` structure should *already* be documented as part of the scene file documentation... but we definitely would do well to have JS examples of modifying the `config` dynamically to make the connection more explicit.